src/components/Banner/Banner.tsx

```diff
@@ -88,47 +88,48 @@ export const Banner = (props: IBannerProps): React.ReactElement => {
 	}
 
 	return (
-		<ReactTransitionGroup
```

Comment:
All I did was replace this component with the two below. The rest of the code is spacing issue.